cinnamonvantage@garlayntoji/files/cinnamonvantage@garlayntoji/applet.js

```diff
@@ -0,0 +1,249 @@
+// This extension was developed by:
+// * Garlayn Toji
+// Version: 1.0.0
+// Date: 2024-08-07
+// License: GPLv2+
+// Copyright (C) 2024-2025 Garlayn Toji
+// Cinnamon Vantage Cinnamon Applet
+
+const Lang = imports.lang;
+const Applet = imports.ui.applet;
+const PopupMenu = imports.ui.popupMenu;
+const Util = imports.misc.util;
+const GLib = imports.gi.GLib;
+const Gio = imports.gi.Gio;
+const Mainloop = imports.mainloop;
+const Settings = imports.ui.settings;
+const Main = imports.ui.main;
+
+const UUID = "cinnamonvantage@garlayntoji";
+
+// Icon location
+const ICON_PATH = GLib.get_home_dir() + '/.local/share/cinnamon/applets/cinnamonvantage@garlayntoji/icon.png';
```

Comment:
There is a way you can retrieve the local applet path that is cleaner. The current approach will work with the caveat that if the upstream path changes, this would break.